apps/website/src/components/layout/navbar/Navbar.tsx

```diff
@@ -23,7 +23,7 @@ export const Navbar = () => {
             {/* Logo */}
             <Link
               href="/"
-              className="flex items-center px-1 md:px-2"
+              className="-ml-4 flex items-center"
```

Comment:
```suggestion
              className="flex items-center md:-ml-4"
```

this breaks on smartphones 
![negative-margin-mobile](https://github.com/alveusgg/alveusgg/assets/684458/c72c82a1-b744-4d48-a412-6cb601f691e7)

Reply:
Oops, completely forgot this is shared across mobile + desktop!